android/src/main/kotlin/io/okhi/flutter/okhi_flutter/OkhiFlutterPlugin.kt

```
@@ -0,0 +1,561 @@
+        }
+    }
+
+    override fun onDetachedFromEngine(binding: FlutterPlugin.FlutterPluginBinding) {
+        channel.setMethodCallHandler(null)
+        closeStream()
+    }
+
+    override fun onAttachedToActivity(binding: ActivityPluginBinding) {
+        try {
+            activity = binding.activity
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)